corehq/apps/analytics/tasks.py

```diff
@@ -163,3 +200,51 @@ def identify(email, properties):
         km = KISSmetrics.Client(key=api_key)
         km.set(email, properties)
         # TODO: Consider adding some error handling for bad/failed requests.
+
+
+@periodic_task(run_every=crontab(minute="0", hour="0"), queue='')
+def track_periodic_data():
+    """
+    Sync data that is neither event or page based with hubspot/Kissmetrics
+    :return:
+    """
+    # Start by getting a list of web users mapped to their domains
+    six_months_ago = date.today() - timedelta(days=180)
+    users_to_domains = UserES().web_users().last_logged_in(gte=six_months_ago).fields(['domains', 'email']).run().hits
+    # users_to_domains is a list of dicts
+
+    domains_to_forms = FormES().terms_facet('domain', 'domain').size(0).run().facets.domain.counts_by_term()
```

Comment:
I initially thought about splitting each of these metrics out into separate functions, but decided this approach would be better below where I only need to iterate through users and domains once.